SPECS/containerized-data-importer/containerized-data-importer.spec

```diff
@@ -32,13 +32,15 @@ BuildRequires:  pkgconfig
 BuildRequires:  rsync
 BuildRequires:  sed
 ExclusiveArch:  x86_64 aarch64
+Provides:       cdi = %{version}-%{release}
```

Comment:
Since this hasn't been release yet, why not just change the package name to cdi?  That seems to be what Fedora offers.  containerized-data-importer appears to be OpenSUSE's package name.

Reply:
That's what Ameya started with, but it's already in preview in PMC, so it would at least require a cherry-pick from `main` or a full merge, plus a restart of the review process or (parts of it). Also, this is a lot smaller change.